### PR TITLE
Score calculations

### PR DESCRIPTION
**Pull request by Bennett567**
https://github.com/online-bridge-hackathon/play-engine/issues/4

### CODE REVIEW

**Comment by Bennett567** [comment]
Does not have team match scoring yet

File: lib/IMP.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+function IMPs(score1, score2) {
+    var diff = Math.abs(score1 - score2); //score 1 is the teams score on table 1, score 2 the same teams score on table 2.
+    var points;
+    if (score1 == score2) {
+        return points= [0, 0];          //first number is IMP of the team whose score we entered, score 2 is the other teams IMP
+
+    }
+    
+        if (diff >= 20 && diff <= 40) {
```
Comment (mbodell) [inline lib/IMP.js:16]: You need to check for diff < 20 otherwise you'll score 3nt+1 versus 4M= as worth 24 imps.

Comment (Bennett567) [reply lib/IMP.js:16]: thanks, fixed

**Comment by Bennett567** [comment]
On the VP calculating file, where we found the algorithm, they said this:
Occasionally because of the rounding the rule V(i+1) - V(i) <= V(i) - V(i-1) will be violated. In that case the value of V(i) is increased by .01. Usually this will eliminate all violations; sometimes it takes as many as four iterations of this procedure to eliminate all of them.

We do not understand this, if anyone does, any help is welcome.

**Pull request merged**